editor/components/inserter/style.scss

```diff
@@ -1,3 +1,7 @@
+$content-height: 250px;
```

Comment:
My understanding is these assigned variables are global to all stylesheets, and this seems like it might be a common name. Maybe worth prefixing to act as namespace if we want the variables?

Reply:
Good point